pegasus/sites.v3/code.org/public/js/volunteer-local.js

```diff
@@ -429,3 +429,25 @@ function i18n(token) {
 
   return labels[token];
 }
+
+/**
+ * Replaces special characters in string by HTML entities.
+ * List of special characters is taken from
+ * https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html
+ *
+ * This is a copy of the same function from @cdo/apps/src/utils.js.
```

Comment:
A note for posterity: Long-term, we'd like to pull this JavaScript out of pegasus and into our Webpack build.  At that point we'll want to remove this copy of `escapeHtml` and import the one from utils.js instead.  This approach is a temporary one to minimize possible impact of this change so close to Hour of Code.

Reply:
Tracked by https://codedotorg.atlassian.net/browse/PLC-654